tests/gocase/unit/cms/cms_test.go

```diff
@@ -0,0 +1,245 @@
+		require.NoError(t, infoRes.Err())
+		infoSlice, ok := infoRes.Val().([]interface{})
+		require.True(t, ok, "Expected cms.info to return a slice")
+
+		infoMap := make(map[string]interface{})
+		for i := 0; i < len(infoSlice); i += 2 {
+			key, ok1 := infoSlice[i].(string)
+			value, ok2 := infoSlice[i+1].(int64)
+			require.True(t, ok1 && ok2, "Expected cms.info keys to be strings and values to be int64")
+			infoMap[key] = value
+		}
+
+		require.Equal(t, int64(2000), infoMap["width"])
+		require.Equal(t, int64(4), infoMap["depth"])
+		require.Equal(t, int64(0), infoMap["count"])
+	})
+
+	t.Run("cms.incrby - Basic Increment Operations", func(t *testing.T) {
+		require.NoError(t, rdb.Do(ctx, "DEL", "cmsA").Err())
+		res := rdb.Do(ctx, "cms.initbydim", "cmsA", "100", "10")
+		require.NoError(t, res.Err())
+		require.Equal(t, "OK", res.Val())
+
+		elements := map[string]string{"apple": "7", "orange": "15", "mango": "3"}
+		for key, count := range elements {
+			res = rdb.Do(ctx, "cms.incrby", "cmsA", key, count)
+			require.NoError(t, res.Err())
+			require.Equal(t, "OK", res.Val())
+		}
+
+		for key, expected := range elements {
+			res = rdb.Do(ctx, "cms.query", "cmsA", key)
+			require.NoError(t, res.Err())
+			countSlice, ok := res.Val().([]interface{})
+			require.True(t, ok, "Expected cms.query to return a slice")
+			require.Len(t, countSlice, 1, "Expected cms.query to return a single count")
+			count, ok := countSlice[0].(string)
+			require.True(t, ok, "Expected count to be a string")
+			require.Equal(t, expected, count, fmt.Sprintf("Count for key '%s' mismatch", key))
+		}
+
+		// Verify total count
+		infoRes := rdb.Do(ctx, "cms.info", "cmsA")
+		require.NoError(t, infoRes.Err())
+		infoSlice, ok := infoRes.Val().([]interface{})
+		require.True(t, ok, "Expected cms.info to return a slice")
+
+		// Convert the slice to a map for easier access
+		infoMap := make(map[string]interface{})
+		for i := 0; i < len(infoSlice); i += 2 {
+			key, ok1 := infoSlice[i].(string)
+			value, ok2 := infoSlice[i+1].(int64)
+			require.True(t, ok1 && ok2, "Expected cms.info keys to be strings and values to be int64")
+			infoMap[key] = value
+		}
+
+		total := int64(0)
+		for _, cntStr := range elements {
+			cnt, err := strconv.ParseInt(cntStr, 10, 64)
+			require.NoError(t, err, "Failed to parse count string to int64")
+			total += cnt
+		}
+		require.Equal(t, total, infoMap["count"], "Total count mismatch")
+	})
+
+	// Increment operation on a non-existent CMS
+	t.Run("cms.incrby - Increment Non-Existent CMS", func(t *testing.T) {
+		res := rdb.Do(ctx, "cms.incrby", "nonexistent_cms", "apple", "5")
+		require.Error(t, res.Err())
+	})
+
+	// Query for non-existent element
+	t.Run("cms.query - Query Non-Existent Element", func(t *testing.T) {
+		require.NoError(t, rdb.Do(ctx, "DEL", "cmsA").Err())
+		res := rdb.Do(ctx, "cms.initbydim", "cmsA", "100", "10")
+		require.NoError(t, res.Err())
+		require.Equal(t, "OK", res.Val())
```

Comment:
Can we also query a non-exists key like "cms.incrby - Increment Non-Existent CMS"